<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMenu</class>
 <widget class="QMainWindow" name="UserMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Mode</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="watchListList"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="movieLabel">
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="text">
         <string>Movie</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="seeWatchListButton">
          <property name="text">
           <string>See watch list </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>Delete movie from watch list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="seeMoviesButton">
          <property name="text">
           <string>See available movies &amp;&amp; update watch list</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" rowspan="2">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="titleLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="genreLabel">
          <property name="text">
           <string>Genre:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="genreLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="trailerLabel">
          <property name="text">
           <string>Trailer:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="trailerLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="yearLabel">
          <property name="text">
           <string>Year of appearance:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="yearLineEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="likesLabel">
          <property name="text">
           <string>Number of likes:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="likesLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="ratingLayout">
          <item>
           <widget class="QLabel" name="ratingLabel">
            <property name="text">
             <string>Did you like this movie?</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="likeRadioButton">
            <property name="text">
             <string>yes!!</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="likeRadioButton_2">
            <property name="text">
             <string>no..</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="thankRating">
          <property name="text">
           <string>Thank you for your rating!</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addToWatchListButton">
          <property name="text">
           <string>Add movie to watch list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="text">
           <string>Next Movie</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
